src/viking.js

```diff
@@ -1,11 +1,83 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    }
+    attack() {return this.strength}
+    receiveDamage(damage) {this.health=this.health-damage; }
+    
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength);
+        this.name = name; 
+    }
+
+    receiveDamage(damage){
+        this.health = this.health-damage;
+    if (this.health > 0){
+        return `${this.name} has received ${damage} points of damage`;
+    } else {
+        return `${this.name} has died in act of combat`;
+    }
```

Comment:
**ES6 Tip**: Since this conditional statement only executes one thing in either scope, you can use a ternary operator here. See suggestion below
```suggestion
    return this.health > 0 ? `${this.name} has received ${damage} points of damage` : `${this.name} has died in act of combat`;
```

It saves you a few lines of code also 👍🏾